xtas/tasks/_nl_conll_ner.py

```diff
@@ -31,12 +31,12 @@ def _download_training_data():
 
     Returns an iterable over the lines of the concatenated dataset.
     """
-    return (ln for part in ["train", "testa", "testb"]
```

Comment:
This gets downloaded for every session now. Maybe it would be better if we made a local copy?